icetea/helper/patricia.js

```diff
@@ -51,15 +52,29 @@ const lastBlock = () => {
   })
 }
 
+const getTxHashes = () => {
+  return new Promise((resolve, reject) => {
+    db.get(txHashesKey, (err, value) => {
+      if (err) {
+        if (err.notFound) {
+          return resolve(null)
+        }
+        return reject(err)
+      }
+      return resolve(serializer.deserialize(value))
```

Comment:
Because hash is base 58, could we use customized deserializer to save some space (available in icetea common)?